Eat_and_Greet/src/screens/OnboardingScreens/emailVerify.tsx

```diff
@@ -20,7 +48,8 @@ function EmailVerify() {
                         Didn’t get a code?
                         Check your spam!
                         </Text>
-                    <RoundedButton name="Verify" height="7%" onPress={() => navigation.navigate('ID Verification')}/> 
+                    <RoundedButton name="Verify" height="7%" onPress={() => emailVerify()}/>
```

Comment:
Maybe switch one to "send verification email" and the other to verify. That way if a user doesn't get the email, they can request another, and the verify button will check if its verified alerting "Success" or "Email Verification not completed".